clash-prelude/clash-prelude.cabal

```diff
@@ -262,6 +263,11 @@ Library
                       Clash.Sized.Internal.Signed
                       Clash.Sized.Internal.Unsigned
 
+                      Clash.Verification
+                      Clash.Verification.DSL
```

Comment:
I wonder whether the DSL combinators should live in their own module :thinking: It seems like more work to just import what should be the nicer API. What about changing it to instead hide the implementation details in a `Clash.Verification.Internal` module?

Reply:
The tick versions of the types should go in an `Internal` module I agree. I put the DSL in a separate module, because it collides with  functions in prelude.

Reply:
Maybe use something like `(:&:)` and `(:|:)` to avoid this (or some other throwing a symbol in to remove the name clash)?

Reply:
`x :& (y :| z)` doesn't look _too_ bad

Reply:
I still have to think about this one :P. I think I'd be alright with `:|:`.